students/17th/team1/ChiohKim/user/views.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from django.views     import View
+from django.http      import HttpResponse, JsonResponse
+from django.db.models import Q
+
+from .models      import Account
+
+class SignupView(View):
+    def post(self, request):
+        signup_data = json.loads(request.body)
+
+        try:
+            if Account.objects.filter(email=signup_data['email']).exists() or Account.objects.filter(phonenumber=signup_data['phonenumber']).exists() or Account.objects.filter(name=signup_data['name']).exists():                  
+
+                return JsonResponse({"message":"EXIST_USER"}, status=400) #에러메세지에도 컨벤션이있나요?
+            if len('password') < 8:
```

Comment:
```suggestion
            if len('password') < MINIMUM_PASSWORD_LENGTH:
```
8 과 같은 비교값의 경우 하드코딩 할 수 도 있지만, 장기적으로 보자면 추후 수정시 직접 로직을 수정하러 와야하는 번거로움이 있죠. 
잘 변하지 않을 기준 값이지만, 여러 곳에서 쓰이게 된다면 추후에는 번거로움이 생깁니다. 이러한 기준값을 상수로 선언해서 관리하는 것을 추천드립니다.

상수 선언은 파일 상단 import 열 하단, 클래스내부에서만 사용될 경우 클래스 선언부 바로 아래, 보안이 중요한 경우, 설정 파일에 선언 후 임포트 하는 방법등을 선택해서 사용할 수 있습니다!

Reply:
수정했습니다!